pkg/build_deb.sh

```diff
@@ -81,13 +81,14 @@ fi
 ##
 
 mkdir -p $ROOT/DEBIAN
-echo "9" >$ROOT/DEBIAN/compat
+#echo "9" >$ROOT/DEBIAN/compat
 cat >$ROOT/DEBIAN/control <<EOF
 Package: php5-suhosin-extension
 Section: php
 Priority: extra
 Maintainer: Ben Fuhrmannek <ben@sektioneins.de>
 Homepage: http://www.suhosin.org/
+Depends: libc6 (>= 2.4), phpapi-20131226, php5-common, ucf
```

Comment:
Added php5-common, which provides phpapi-... and already depends on ucf and libc. This should work for all debian/ubuntu versions, even wheezy.